pkg/sink/sink.go

```diff
@@ -62,6 +62,13 @@ type Sink struct {
 	Logger                 *zap.SugaredLogger
 }
 
+// Response defines the HTTP body that the Sink responds to events with.
+type Response struct {
+	EventListener string `json:"EventListener"`
+	Namespace     string `json:"namespace,omitempty"`
+	EventID       string `json:"eventID,omitempty"`
```

Comment:
could these fields have docstrings as well? e.g. i can't tell what kind of values ot expect in `EventListener` just looking at this